no-computer.md

```diff
@@ -1,44 +1,52 @@
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This "DSL" is appropriate because all violin players use the same "DSL". Therefore, when players play the same piece, they are similar in sound. However, this "DSL" also has flexibility in that there are many ways to play the same piece. Though the notes may be exactly the same, the "DSL" allows the user to express their personal preference and tastes which is reflected in their sound.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Growing up, learning violin was extremely frustrating. The concept of having to train the ear to recognize pitches in order to properly play the correct notes was baffling to me: I often complained about how the piano had discrete notes while the violin did not. This idea is exciting because a) I now regret having quit violin after maturing and recognizing how fun of a hobby instruments can be and b) so no one has to go through the struggle I did.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Violin is a hard instrument
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would give some placement of their fingers which would then produce some sound. Because the domain is extremely specific, this is pretty much the only way to interact with the problem domain, and therefore the right way.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When the user places their fingers on the fingerboard, they use a bow with their other hand to vibrate the strings and produce a note. The program interacts with the user based off of the note they hear. They correlate this note with their finger pattern. Had they misplaced their fingers (which is very likely), they will produce the wrong note and will correct appropriately.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+*Nothing* is easy in this language. It should be possible to play correct notes based off of finger placement. Producing music from stringing together notes based off of various finger palcements is also possible. When the musical piece becomes more difficult e.g. requiring quick and accurate finger placement, playing the correct notes gets progressively harder. Given that there are only 4 strings and all cannot be played at the same time, it should be impossible to play 5+ note chords, which is possible on the piano.  
+
```

Comment:
Maybe some standard finger placements would be easier than others? I like how you say that it's impossible to play 5+ note chords because you're taking the physical limitations of the interface into account.